docs/codeflow/using-pr-new.md

```diff
@@ -4,19 +4,17 @@ title: Using pr.new
 
 # {{ $frontmatter.title }}
 
-Open Source contributions have never been easier 🥰 
-
 This page covers using pr.new to open, view, and contribute to any project through our Codeflow IDE or Web Publisher.
 
+Built to make open source contributions as easy as possible. 🥰
+
 ## What is pr.new?
 
 <!--@include: ./parts/pr-new.md-->
 
 ## How does pr.new work?
 
-This short URL enables magical experiences by choosing just the right tools for the job.
-
-Below you will learn what to expect in different scenarios.
+Behind the scenes, this short URL will choose different tools in different scenarios:
```

Comment:
Shortened as much as possible. Tried to still be descriptive, but also allow the next sections to "do the talking".